docs/source/science/calculators/mobility.rst

```diff
@@ -1,7 +1,7 @@
 .. _science_calculators_mobility:
 
-Electron and Hole Mobility. Calculator **mobility**.
-===================================================
+Electron and Hole Mobility
```

Comment:
Good point. There is currently no name for the production calculator. "mobility" is still there but outdated. I can call it as you wish. I.e. Electron_and_hole_Mobility.

Reply:
I don't mind, just remember that it is a string that is displayed directly to the user as is. So I would not include any "_" or anything. You can also just call it "Mobility Calculator" or "Charge Carrier Mobility" - Or you call them "Nanomatch Multiscale Mobility" and "Nanomatch Multiscale Morphology"

Reply:
I have removed the "Calculator **mobility**" because it looked ugly as I cited this page.
In the present case, the name of the link does fits in the text flow if referred from the outside like in the property description.

Reply:
Ok, just make sure to update the Calculator in the DiaDEM database according to the string you use in the title. Talk to Loveleen if this is possible. If now, we may need to delete the calculator and the properties that refer to it, and then re-populate. The lables in the frontend and the documentation should be in sync. If this was unclear anywhere along the workpath of your recent weeks, please clarify in the respective parts of the documentation (database curation / instructions for the documentation).